tx-source/site_core.php

```diff
@@ -1166,6 +1166,7 @@
 $Definition['Previous'] = 'Previous';
 $Definition['Privacy'] = 'Privacy';
 $Definition['Privacy Settings'] = 'Privacy Settings';
+$Definition['<0>Privacy Policy</0> and '] = '<0>Privacy Policy</0> and ';
```

Comment:
Don't split this string in 2. It's possible the order in another language does not match the way we have it here. You should be writing it like this:

`<0>Privacy Policy</0> and <1>Terms of Service</1>`